src/lib/swapper/swappers/ThorchainSwapper/buildThorTrade/buildThorTrade.ts

```diff
@@ -111,6 +113,8 @@ export const buildTrade = async (
     if (maybeThorTxInfo.isErr()) return Err(maybeThorTxInfo.unwrapErr())
     const { vault, opReturnData } = maybeThorTxInfo.unwrap()
 
+    // TODO(gomes): it is not - plumb from through
```

Comment:
Actual TODO, to be akschually tackled in a follow-up, reduces the risk area like so